lambdas/db-migration/tests/test-index.js

```diff
@@ -0,0 +1,46 @@
+const test = require('ava');
+const Knex = require('knex');
+const cryptoRandomString = require('crypto-random-string');
+
+const { handler } = require('../dist/lambda');
+
+const knex = Knex({
+  client: 'pg',
+  connection: {
+    host: 'localhost',
+    user: 'postgres',
+    password: 'password',
+    database: 'postgres',
+  },
+});
+
+test.before(async () => {
+  await handler({
+    env: {
+      KNEX_ASYNC_STACK_TRACES: 'false',
+      KNEX_DEBUG: 'false',
+      PG_HOST: 'localhost',
+      PG_USER: 'postgres',
+      PG_PASSWORD: 'password',
+      PG_DATABASE: 'postgres',
+    },
+  });
+});
+
+test.after.always(async () => {
+  await knex.destroy();
+});
+
+test('collections table enforces unique constraint on name/version', async (t) => {
```

Comment:
This test isn't really meaningful and can probably go away. That being said, what tests (if any) should we have to verify the schema migration applied as expected?

Reply:
I'm inclined to say "none", because the code that's using the DB is going to fail if an expected DB migration didn't work.